utils/grpc/client_test.go

```diff
@@ -72,6 +58,7 @@ func TestParse(t *testing.T) {
 		{in: "http://btfs.io:2333", out: &parsedURL{schema: "http", host: "btfs.io", port: 2333}},
 		{in: "https://127.0.0.1", out: &parsedURL{schema: "https", host: "127.0.0.1", port: 443}},
 		{in: "http://127.0.0.1", out: &parsedURL{schema: "http", host: "127.0.0.1", port: 80}},
+		{in: "hub-service:8080", out: &parsedURL{schema: "http", host: "hub-service", port: 8080}},
```

Comment:
what is a hub-service:8080 ? Is this related to kubernetes services? @laipogo

Reply:
> what is a hub-service:8080 ? Is this related to kubernetes services? @laipogo

nvm,  it's just a test case. please ignore it.